mojaloop/Chart.yaml

```diff
@@ -127,6 +127,31 @@ dependencies:
     version: ">= 15.6.0"
     repository: "file://../ml-testing-toolkit-cli"
     condition: ml-ttk-test-val-sdk-r2p.tests.enabled
+  - name: ml-testing-toolkit-cli
+    alias: ml-ttk-test-setup-fx
+    version: ">= 15.6.0"
+    repository: "file://../ml-testing-toolkit-cli"
+    condition: ml-ttk-test-setup-fx.tests.enabled
+  - name: ml-testing-toolkit-cli
+    alias: ml-ttk-test-val-fx
+    version: ">= 15.6.0"
+    repository: "file://../ml-testing-toolkit-cli"
+    condition: ml-ttk-test-val-fx.tests.enabled
+  - name: ml-testing-toolkit-cli
+    alias: ml-ttk-test-setup-proxy
+    version: ">= 15.6.0"
+    repository: "file://../ml-testing-toolkit-cli"
+    condition: ml-ttk-test-setup-proxy.tests.enabled
+  - name: ml-testing-toolkit-cli
+    alias: ml-ttk-test-val-payer-proxy
+    version: ">= 15.6.0"
+    repository: "file://../ml-testing-toolkit-cli"
+    condition: ml-ttk-test-val-payer-proxy.tests.enabled
+  - name: ml-testing-toolkit-cli
+    alias: ml-ttk-test-val-payee-proxy
+    version: ">= 15.6.0"
+    repository: "file://../ml-testing-toolkit-cli"
+    condition: ml-ttk-test-val-payee-proxy.tests.enabled
   - name: ml-testing-toolkit-cli
```

Comment:
Do we need separate test jobs for proxy and fx? I thought we are going to have it in the same GP and provisioning collections.

Reply:
They are currently separated.